src/layers/BingLayer.js

```diff
@@ -42,9 +42,10 @@ class BingLayer extends Layer {
 
         this.createLayer()
 
-        super.addTo(map)
+        await super.addTo(map)
```

Comment:
I assume this was a bug before?

Reply:
Yes, affected the update of the Bing Maps attribution.